### ISSUE TITLE
Vspipe crash

### ISSUE THREAD

**Issue by Yohan050** [vapoursynth/vapoursynth#316]
Since a month , I've fot issues with vspipe and yolkrCR script. When i'd like to preview the script with vsedit or want to encode the video with parameters as Supersampling = -1, vsedit and vspipe are crashing and  "pipe:: Invalid data found when processing input" is written.
I tried everything like reinstalling vapoursynth or python or update all dependences, error always coming back. As on Debian, error is the same. Are you aware of this issue ? I had a talk with YamashitaRen  who developed YoloCR but he couldn't understand. Do you have an idea, regards ?
I link some screenshot I have make during my tests, the script and the video I used.

http://puu.sh/wm8AW.png
http://puu.sh/wl711.png
http://puu.sh/wm8Fd.vpy
https://zerofansub.net/ddl/[Zero]Kujibiki_Unbalance_2[HD][H264-AAC]/[Zero]Kujibiki_Unbalance[04v2][HD][H264-AAC].mp4

**Comment by dubhater** [comment]
To me it looks like ffmpeg chokes on the first frame, so let's examine that:
```sh
vspipe -y YoloCR.vpy /tmp/frame.y4m --end 0
```
If that command completes without any errors, open frame.y4m in a hex editor. If you don't see anything weird, upload it.

**Comment by Yohan050** [comment]
Hi dubhater.
I used your command line but it returns with this error : "Failed to open output for writing" and any files haven't be created.

I changed the path by removing the /tmp/  to keep the filename only and the command started but vspipe.exe crashed after creating frame.y4m but the file is empty. On Debian, I have the error : "Failed to initialize Vapoursynth environment. 

http://puu.sh/wmXDA.y4m
